### ISSUE TITLE
"Editor has crashed"

### ISSUE THREAD

**Issue by crustle** [atom/atom#13743]
### Prerequisites

* [ X] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
    * Followed all applicable steps in the debugging guide: http://flight-manual.atom.io/hacking-atom/sections/debugging/
    * Checked the FAQs on the message board for common solutions: https://discuss.atom.io/c/faq
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom
    * Checked that there is not already an Atom package that provides the described functionality: https://atom.io/packages

### Description

Error message "The editor has crashed" with options to close window, reload, keep window open.

### Steps to Reproduce

1. Open from command line or from Ubuntu launcher
2. 
3. 

**Expected behavior:** [What you expect to happen]

Atom opens.

**Actual behavior:** [What actually happens]

Atom crashes.

**Reproduces how often:** [What percentage of the time does it reproduce?]

100%

### Versions
Atom    : 1.13.1
Electron: 1.3.13
Chrome  : 52.0.2743.82
Node    : 6.5.0apm  1.14.1
npm  3.10.5
node 4.4.5
python 2.7.6
git 1.9.1

### Additional Information

I have tried launching --safe and --clear-window-state to no avail.  Uninstalled then reinstalled after rebooting my computer, also with no change.

Note: I powered down my computer while atom was still running previous to this crash.  :/

**Comment by 50Wliu** [comment]
Does renaming the `~/.atom` folder to `~/.atom-backup` help?

**Comment by mbalmer** [comment]
Same issue here on Fedora 25, updated as of today.

**Comment by mbalmer** [comment]
Moving away ~/.atom actually makes the editor work again, but all settings are lost, so that not really a cool solution...

**Comment by mbalmer** [comment]
I removed ~/.atom/blob-store and it worked again.  No idea what blob-store is good for, though.

**Comment by crustle** [comment]
Yes, renaming ~/.atom to ~/.atom-backup worked.  Thank you!

**Comment by kreck** [comment]
same issue on MacOS Sierra 10.12.2 with same specs (atom, electron, node, etc.) as @crustle . However removing  ~/.atom doesn't not help in my case. The Editor still crashes when i open atom.

**Comment by Dundee** [comment]
Same issue on Archlinux with Atom 1.14.1. When I move ~/.atom I at least get error message:

```
[2342:0213/132916:INFO:CONSOLE(61)] "Error: Could not create an input method.
    at Error (native)
    at Object.<anonymous> (/usr/lib/atom/node_modules/keyboard-layout/lib/keyboard-layout.js:9:17)
    at Module._compile (/usr/lib/atom/src/native-compile-cache.js:109:30)
    at Object.value [as .js] (/usr/lib/atom/src/compile-cache.js:216:21)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (/usr/lib/atom/src/native-compile-cache.js:50:27)
    at Object.<anonymous> (/usr/lib/atom/node_modules/atom-keymap/lib/helpers.js:6:20)
    at Object.<anonymous> (/usr/lib/atom/node_modules/atom-keymap/lib/helpers.js:372:4)
    at Module._compile (/usr/lib/atom/src/native-compile-cache.js:109:30)
    at Object.value [as .js] (/usr/lib/atom/src/compile-cache.js:216:21)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (/usr/lib/atom/src/native-compile-cache.js:50:27)
    at Object.<anonymous> (/usr/lib/atom/node_modules/atom-keymap/lib/key-binding.js:4:10)
    at Object.<anonymous> (/usr/lib/atom/node_modules/atom-keymap/lib/key-binding.js:129:4)
    at Module._compile (/usr/lib/atom/src/native-compile-cache.js:109:30)
    at Object.value [as .js] (/usr/lib/atom/src/compile-cache.js:216:21)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (/usr/lib/atom/src/native-compile-cache.js:50:27)
    at Object.<anonymous> (/usr/lib/atom/node_modules/atom-keymap/lib/keymap-manager.js:17:11)
    at Object.<anonymous> (/usr/lib/atom/node_modules/atom-keymap/lib/keymap-manager.js:639:4)
    at Module._compile (/usr/lib/atom/src/native-compile-cache.js:109:30)", source: file:///usr/lib/atom/static/index.js (61)
Failed to get crash dump id.
Report Id: 
Gtk-Message: GtkDialog mapped without a transient parent. This is discouraged.
```

**Comment by aboettger-tuhh** [comment]
```
$ atom -f --safe
[8882:0213/133902:ERROR:browser_main_loop.cc(231)] Running without the SUID sandbox! See https://chromium.googlesource.com/chromium/src/+/master/docs/linux_suid_sandbox_development.md for more information on developing with the sandbox on.
Gtk-Message: (for origin information, set GTK_DEBUG): failed to retrieve property `gtk-primary-button-warps-slider' of type `gboolean' from rc file value "((GString*) 0x25357e382280)" of type `gboolean'
Failed to get crash dump id.
Report Id:
```

Removing `~/.atom` doesn't help

**Comment by jgeboski** [comment]
I am running into this after doing a system update. I upgraded to `electron-1.3.13` and `atom-1.13.1`. I have tried reverting both electron and atom to known working version (`electron-1.3.6` and `atom-1.12.7`), but this still results in the startup error.

```
Program terminated with signal SIGILL, Illegal instruction.
#0  0x00000000031e2b50 in WTF::partitionAllocInit(WTF::PartitionRoot*, unsigned long, unsigned long) ()
#1  0x0000000000da62a0 in blink::FreePagePool::takeFreePage(int) ()
#2  0x00007ffd9dabdbf8 in ?? ()
#3  0x00007ffd9dabdc58 in ?? ()
#4  0x000000a9a7680380 in ?? ()
#5  0x0000000000020000 in ?? ()
#6  0x000000a9a7618168 in ?? ()
#7  0x000000a9a760b9e0 in ?? ()
#8  0x0000000000000000 in ?? ()
```

**Comment by 50Wliu** [comment]
@Dundee your issue has been fixed in Atom 1.14.2, which may be released sometime this week.

@aboettger-tuhh, @jgeboski please open separate issues.

Since the original issue has been solved, I'm closing this.
